projects/RabbitMQ.Client/client/api/ConnectionFactory.cs

```diff
@@ -175,6 +175,12 @@ public sealed class ConnectionFactory : ConnectionFactoryBase, IAsyncConnectionF
         /// </summary>
         public bool DispatchConsumersAsync { get; set; } = false;
 
+        /// <summary>
+        /// TODO.
+        /// Defaults to 1.
+        /// </summary>
+        public int AsyncConsumerConcurrency { get; set; } = 1;
```

Comment:
This is just a working name. Could be anything that is more meaningful.

Reply:
Would be good to set the description to clearly say that it defines the parallelism of the consumer processing, perhaps providing an example of setting it to Environment.ProcessorCount?

Reply:
I force pushed a more thorough explanation. Have a look